docs/scenarios_authoring/Enrichers.md

```diff
@@ -41,6 +41,28 @@ DatabaseLookupEnricher is a specialized look-up component; it returns all column
 
 ![alt_text](img/databaseLookupEnricher.png "databaseLookup Enricher")
 
+### DecisionTable enricher
+
+This component allows the user to match input to a pre-defined result. Please be wary that this component works a bit different than a classical decision table([Decision tables in Wikipedia](https://en.wikipedia.org/wiki/Decision_table)). It consists of the fields:
+- Name: the name of the component.
+- Decision Table: a user defined table of type TabularTypedData, discussed below.
```

Comment:
TBH `TabularTypedData` is our internal type which leaked into UI (I reported this fact but at the moment there is a decision to not bother with this). We should assume that users don't have to know (and the do NOT know) what the type is. 

Let's describe that:
1. the table could have many columns
2. each column has a type
3. the cells are validated
4. no value in a cell means `null`